pkg/model/components/cilium.go

```diff
@@ -168,5 +168,34 @@ func (b *CiliumOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
+	ingress := c.Ingress
+	if ingress != nil {
+		if ingress.Enabled == nil {
+			ingress.Enabled = fi.PtrTo(true)
+		}
+		if ingress.EnforceHttps == nil {
+			ingress.EnforceHttps = fi.PtrTo(true)
+		}
+		if ingress.EnableSecretsSync == nil {
+			ingress.EnableSecretsSync = fi.PtrTo(true)
+		}
+		if ingress.SecretsNamespace == "" {
+			ingress.SecretsNamespace = "cilium-secrets"
```

Comment:
We tend to reuse `kube-system` for things like this as it's the one namespace kOps owns.

Reply:
Sure, we can use `kube-system` - I just used the default namespace provided in the [cilium helm chart](https://github.com/cilium/cilium/blob/v1.13.6/install/kubernetes/cilium/values.yaml#L616). I'd just go the kOps way then :+1: